repos/fdbt-netex-output/src/data/auroradb.ts

```diff
@@ -77,7 +77,39 @@ export const getOperatorDataByNocCode = async (nocCodes: string[]): Promise<Oper
             throw new Error(`No operator data found for nocCodes: ${cleansedNocs.join(',')}`);
         }
 
-        return operatorData;
+        const allNocCodes: string[] = [];
+        let nocCount: { [key: string]: number } = {};
```

Comment:
When we have an object like this, its always better to not use 'key', because that doesnt help the person reading it to understand what the keys of the object will be, only that they will be a string.

`{ [nocCode: string]: number }` would be more useful, because its the nocCode that we use.